File: CPP/Program/PetShop.cpp
#include <iostream>
#include <string>
using namespace std;

class PetShop {
protected:
    string id;
    string nama_produk;
    double harga_produk;
    int stok_produk;

public:
    PetShop(string id, string nama_produk, double harga_produk, int stok_produk) {
        this->id = id;
        this->nama_produk = nama_produk;
        this->harga_produk = harga_produk;
        this->stok_produk = stok_produk;
    }

    ~PetShop() {}

    string get_id() {
        return this->id;
    }

    string get_nama_produk() {
        return this->nama_produk;
    }

    double get_harga_produk() {
        return this->harga_produk;
    }

    int get_stok_produk() {
        return this->stok_produk;
    }

    void set_id(string id) {
        this->id = id;
    }

    void set_nama_produk(string nama_produk) {
        this->nama_produk = nama_produk;
    }

    void set_harga_produk(double harga_produk) {
        this->harga_produk = harga_produk;
    }

    void set_stok_produk(int stok_produk) {
        this->stok_produk = stok_produk;
    }
};

File: Python/Program/PetShop.py
class PetShop:
    def __init__(self, id, nama_produk, harga_produk, stok_produk):
        self.id = id
        self.nama_produk = nama_produk
        self.harga_produk = harga_produk
        self.stok_produk = stok_produk
    
    def get_id(self):
        return self.id
    
    def get_nama_produk(self):
        return self.nama_produk
    
    def get_harga_produk(self):
        return self.harga_produk
    
    def get_stok_produk(self):
        return self.stok_produk
    
    def set_id(self, id):
        self.id = id
    
    def set_nama_produk(self, nama_produk):
        self.nama_produk = nama_produk
    
    def set_harga_produk(self, harga_produk):
        self.harga_produk = harga_produk
    
    def set_stok_produk(self, stok_produk):
        self.stok_produk = stok_produk